CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * CardPayments
   * Add `liabilityShift` property to `CardResult`
   * Callback `PayPalSDKError` when `CardClient#approveOrder()` 3DS verification fails
+* Breaking Changes
```

Comment:
Is this a breaking change? If so, it would need to be in a major version branch. It doesn't seem like it would break anything for merchants though so not sure it should be considered that.

Reply:
There was some discussion about this too in Stephanie's iOS PR ([here](https://github.com/paypal/paypal-ios/pull/237#issuecomment-1883619367)). The consensus there was to treat it like a "bug fix to meet brand guidelines" on V1, but please do chime in your thoughts!

Reply:
Done: https://github.com/paypal/paypal-android/pull/232/commits/e687c52f20a59d68e28e1b031a628749eb249cca